androidApp/src/main/java/com/m3u/androidApp/ui/AppNavHost.kt

```diff
@@ -83,7 +80,6 @@ fun AppNavHost(
             },
             navigateToSettingPlaylistManagement = {
                 navigateToRoot(Destination.Root.Setting)
-                Events.settingDestination = eventOf(SettingDestination.Playlists)
```

Comment:
Why remove this

Reply:
i dont think i even edited anything here lol